### ISSUE TITLE
Package request: NetTraffic or similar alternative

### ISSUE THREAD

**Issue by regystro** [lukesampson/scoop#2634]
Hi. Is it possible to add a bandwidth monitor like [NetTraffic](https://www.venea.net/web/nettraffic) or similar alternatives?

Thank you

**Comment by Ash258** [comment]
Here is how manifest could look like. But there is no way to download zip files directly, not using their file proxy. Until someone figure out a way how to get direct URL it's not possible to download file using scoop (System.Net.WebClient).

```json
{
    "version": "1.51.1",
    "description": "Essential network bandwidth monitor",
    "homepage": "https://www.venea.net/web/nettraffic",
    "license": {
        "identifier": "Freeware",
        "url": "https://www.venea.net/web/nettraffic#license"
    },
    "url": "",
    "hash": "",
    "extract_dir": "NetTraffic",
    "persist": "Data",
    "bin": "NetTraffic.exe",
    "shortcuts": [
        [
            "NetTraffic.exe",
            "NetTraffic"
        ]
    ],
    "checkver": {
        "url": "https://www.venea.net/web/downloads_start&id=14",
        "regex": "\\s+Version: ([\\d\\.]+)"
    },
    "autoupdate": {
        "url": "",
        "hash": {
            "url": "https://www.venea.net/web/downloads_start&id=14",
            "regex": "md5\\):\\s([A-Fa-f\\d]{32})"
        }
    }
}
```

**Comment by r15ch13** [comment]
The hash values from the `direct` query parameter are predictable per session:

```
816b112c6105b3ebd537828a39af4818:401
69cb3ea317a32c4e6143e665fdb20b14:402
4f4adcbf8c6f66dcfc8a3282ac2bf10a:404
8cb22bdd0b7ba1ab13d742e22eed8da2:406
0d0fd7c6e093f7b804fa0150b875b868:408
6693a7aacb8a2e74c6f5fe91bc39864b:4010
bd3ef5c19067fe179f71c8b86ea4b39b:4012
1cdf14d1e3699d61d237cf76ce1c2dca:4014
f04cd7399b2b0128970efb6d20b5c551:4016
455cb2657aaa59e32fad80cb0b65b9dc:4018
1192265fb31ab52a6332c561ee6e5258:4020
d5ff135377d39f1de7372c95c74dd962:4022
31b91e3a8737fd8dd3d4b0c8c679049b:4024
c952ce98517ac529c60744ac28364b03:4026
a09e75c5c86a7bf6582d2b4d75aad615:4028
084a8a9aa8cced9175bd07bc44998e75:4030
3cc578f087ea520aefaaf92c802c1a34:4032
...
```

Running `curl` without keeping the cookie will always return `816b112c6105b3ebd537828a39af4818`.
If the session cookie is saved and send back it starts incrementing. This can be repeated for each new session.
So the first valid hash is always `69cb3ea317a32c4e6143e665fdb20b14`.

**Comment by regystro** [comment]
Closing, since it's done: https://github.com/lukesampson/scoop-extras/issues/3460.

Thank you!!

**Comment by rasa** [comment]
Fixed with lukesampson/scoop-extras@4a6b6817b55aa1cc9ce3fbbdb56d662c4acce850

**Comment by rasa** [comment]
Reopened by accident.
